Bug bounty page (#131)
matter-labs/zksync-docs: docs/faq/security.md

## Security audits

[Security audits](/updates/security-audits.md) are being conducted before each major update.

## Bug Bounty Program

> Besides security audits, we offer bug bounty program. You can read more about it [here](/dev/bug-bounty).

Reviewer: ```suggestion
Besides security audits, we have a Bug Bounty program. You can read about it [here](/dev/bug-bounty).
```

Author: In private conversation it was decided that it is hard to determine which is the right way to do it, so I'll keep it as it is
